Jenkinsfile

```diff
@@ -1,10 +1,12 @@
 library 'pipeline-utils@master'
 
+CCV = ""
+CURR_TIMESTAMP = ""
```

Comment:
you can remove this CURR_TIMESTAMP, I think I was using it a while back but now it's just a local variable below, so the global is unnecessary